packages/explorerkit-server/package.json

```diff
@@ -41,7 +41,9 @@
     "vitest": "^1.6.0"
   },
   "dependencies": {
+    "@coral-xyz/anchor": "^0.29.0",
     "@solana/web3.js": "^1.87.2",
+    "rpc-websockets": "7.11.0",
```

Comment:
This is a peer dependency of solana/web3.js that has to be explicitly defined, overwise pnpm install fails